colossalai/auto_parallel/tensor_shard/node_handler/strategy/batch_norm_generator.py

```diff
@@ -76,23 +78,23 @@ def update_memory_cost(self, strategy: ShardingStrategy):
         # compute fwd cost incurred
         # fwd_cost = input + other + bias + output
         fwd_activation_cost = sum([v for k, v in forward_size_mapping.items() if not self.is_param(k)])
-        fwd_parameter_cost = sum([v for k, v in forward_size_mapping.items() if self.is_param(k)])
-        fwd_mem_cost = MemoryCost(activation=fwd_activation_cost, parameter=fwd_parameter_cost)
+        fwd_parameter_and_buffer_cost = sum([v for k, v in forward_size_mapping.items() if self.is_param(k)])
+        fwd_mem_cost = MemoryCost(activation=fwd_activation_cost, parameter=fwd_parameter_and_buffer_cost)
```

Comment:
Since we differentiate parameter and buffer in the `OperationDataType`, we should separate them in the `MemoryCost` as well.

Reply:
Okay.